ledger/ledger-utxo/src/test/java/net/corda/v5/ledger/utxo/transaction/UtxoFilteredTransactionJavaApiTest.java

```diff
@@ -0,0 +1,66 @@
+package net.corda.v5.ledger.utxo.transaction;
+
+import net.corda.v5.crypto.SecureHash;
+import net.corda.v5.ledger.utxo.Command;
+import net.corda.v5.ledger.utxo.StateAndRef;
+import net.corda.v5.ledger.utxo.StateRef;
+import net.corda.v5.ledger.utxo.TimeWindow;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+public class UtxoFilteredTransactionJavaApiTest {
+
+    protected final SecureHash hash = SecureHash.parse("SHA256:0000000000000000000000000000000000000000000000000000000000000000");
+
+    @Test
+    void canReadFilteredTransaction() {
+        // Set up Filtered TX mock
+        UtxoFilteredTransaction filteredTx = Mockito.mock(UtxoFilteredTransaction.class);
+
+        // Notary, time window commands
+        Mockito.when(filteredTx.getNotary()).thenReturn(null);
```

Comment:
For the notary use case, it would probably make sense to set this - no point in hiding the notary from itself, and it potentially allows a bit of extra protection (ensuring that the notary in question is the intended recipient)